Tests/WebSocketKitTests/WebSocketKitTests.swift

```diff
@@ -279,42 +287,6 @@ final class WebSocketKitTests: XCTestCase {
         try server.close(mode: .all).wait()
     }
 
-    func testLocally() throws {
```

Comment:
Why was this removed?

Reply:
I had thought it was something I had added to test locally. It hangs forever the only test that didnt ever pass .

Reply:
Just adding what I mentioned in discord that I removed it because it was skipped anyway and mistakenly thought had made it due to its vague name as I do that when drafting a test. Especially since not skipped it would hang forever.
Adding it back just adds a skipped taste is the takeaway I htink.